src/cv/yolo_video_detect.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import imutils
+import time
+import cv2
+import os
+
+def get_configs(args):
+    # Loads labels
+    labelsPath = os.path.sep.join([args["yolo"], "coco.names"])
```

Comment:
Retrieve this file from the environment variable

Reply:
changes made